tv_apps/smart-home/js/edit.js

```diff
@@ -45,9 +45,11 @@
         this.mainSection.dataset.mode = '';
         this.spatialNavigator.multiAdd(this.regularNavElements);
         this.spatialNavigator.multiRemove(this.editNavElements);
-        this.spatialNavigator.focus(this.editButton);
         this.cardScrollable.setScale();
-        this.cardManager.writeCardlistInCardStore({cleanEmptyFolder: true});
```

Comment:
I would suggest adding some comment for it:
```
// Since writeCardlistInCardStore triggers card-removed event that causes re-focus on card list elements,
// we need to wait until those actions to be done before focusing editButton.
```